testprojects/tests/scala/org/pantsbuild/testproject/exports/BUILD

```diff
@@ -25,3 +25,23 @@ scala_library(
   dependencies=[':C'],
   strict_deps=True,
 )
+
+target(
```

Comment:
Yeah it's in tests/python/pants_test/backend/jvm/tasks/jvm_compile/test_dep_exports_integration.py.

The test method does "list ::" in this dir, thus the new targets are automatically covered.